pallets/creditcoin/src/tests.rs

```diff
@@ -251,19 +291,29 @@ impl TestInfo {
 	}
 }
 
+pub fn get_register_address_message(who: AccountId) -> [u8; 32] {
+	sp_io::hashing::sha2_256(who.encode().as_slice())
+}
+
 #[test]
 fn register_address_basic() {
 	ExtBuilder::default().build_and_execute(|| {
-		let acct: AccountId = AccountId::new([0; 32]);
+		let (key_pair, _) = sp_core::ecdsa::Pair::generate();
+		let pkey = key_pair.public();
+		let signer: MultiSigner = pkey.clone().into();
+		let who = signer.into_account();
+		let message = get_register_address_message(who.clone());
+		let signature = key_pair.sign(message.as_slice());
+		let address = Etherlike::from_public(&pkey);
```

Comment:
Seems like this sequence is duplicated a fair bit, might be nice to factor it out somehow. (I know there's already a lot of duplication in the tests, but I want to try to at least avoid adding more)